code/espurna/ws.ino

```diff
@@ -233,7 +233,9 @@ void _wsParse(AsyncWebSocketClient *client, uint8_t * payload, size_t length) {
     const char* action = root["action"];
     if (action) {
 
-        DEBUG_MSG_P(PSTR("[WEBSOCKET] Requested action: %s\n"), action);
+        if (strcmp(action, "ping") != 0) {
+            DEBUG_MSG_P(PSTR("[WEBSOCKET] Requested action: %s\n"), action);
+        }
```

Comment:
Why not
```cpp
+        if (strcmp(action, "ping") == 0) {
+            wsSend_P(client_id, PSTR("{\"pong\": 1}"));
+            return;
+        }
+
```
just once, right here?

Reply:
The first one is because i didn't want to spam the debug log after each ping-pong (it only prints actions that are not ping...)

Reply:
I mean, check action before print and return.

Reply:
fixed it:)